lib/engine-jit/src/unwind/mod.rs

```diff
@@ -1,3 +1,20 @@
+use wasmer_compiler::CompiledFunctionUnwindInfo;
+
+pub trait UnwindRegistryExt {
```

Comment:
I'm not convinced we need a trait, only one implementation would be needed at runtime (or compiled in the code).

Reply:
We can use `impl  Trait` for this if the concern is correctness and efficiency.  If the concern is over abstraction though, then that  doesn't fix it

Reply:
The concern is correctness. It declares an interface that all unwind registry implementations must fulfill.